test/controllers/ArrivalsControllerSpec.scala

```diff
@@ -76,36 +101,79 @@ class ArrivalsControllerSpec extends FreeSpec with MustMatchers with GuiceOneApp
          </CUSOFFPREOFFRES>
        </CC007A>
      )
-     val result = route(app, request).value
+     val result = TestArrivalsController.createArrivalNotification().apply(request)
```

Comment:
In each of the tests, we should replace this line with `val result = route(app, request).value`